java/HealthVault Mobile/HVSample/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,237 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This layout uses lots of nested linear layouts. this can be bad for performance especially when using the layout:weight property. Consider moving to a relative layout or the more modern constraint layout.